cpp/src/io/utilities/parsing_utils.cuh

```diff
@@ -670,13 +597,15 @@ struct ConvertFunctor {
                                                       parse_options_view const& opts,
                                                       bool as_hex)
   {
+    // TODO what's invalid input
+    bool all_digits_valid = true;
     static_cast<device_storage_type_t<T>*>(out_buffer)[row] =
-      [&opts, output_type, begin, end]() -> device_storage_type_t<T> {
+      [&opts, &all_digits_valid, output_type, begin, end]() -> device_storage_type_t<T> {
```

Comment:
Should `all_digits_valid` be getting used somewhere? I'm not seeing anything that could set it to false right now.

Reply:
It's not used now. It should be used when we decide what is a parsing error for decimal, timestamp and duration types. Right now, it is not decided, just a placeholder. Should I replace it with `true`?

Reply:
```suggestion
      [&opts, output_type, begin, end]() -> device_storage_type_t<T> {
```